src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -217,26 +226,64 @@ std::unique_ptr<data_iterator> drawing_classifier::create_iterator(
 
   data_params.is_train = is_train;
   data_params.target_column_name = read_state<flex_string>("target");
-  data_params.feature_column_name = read_state<flex_string>("feature");
+  const flex_list &features_values = read_state<flex_list>("features");
+  DASSERT_GE(features_values.size(), 1);
+  data_params.feature_column_name = features_values[0].get<flex_string>();
+  /** Until there is only one feature column name, we will read the features
+   *  flex list and record the 0'th index as the feature column name.
+   */
   return create_iterator(data_params);
 }
 
+float_array_map drawing_classifier::get_model_params() const {
+
+  float_array_map raw_model_params = nn_spec_->export_params_view();
+
+  // Strip the substring "_fwd" from any parameter names, for compatibility with
+  // the compute backend. (We preserve the substring in nn_spec_ for inclusion
```

Comment:
why `_fwd` maters?

Reply:
Similar to OD/AC, the Python TF code is written with the assumption that the layer names do not have the `_fwd` string.

Reply:
Could we just remove the `_fwd` from the model spec then?

Reply:
we can remove it, easily.